.github/workflows/docker-ros.yml

```diff
@@ -4,27 +4,57 @@ on: push
 
 jobs:
 
-  ros:
+  ros-noetic:
     runs-on: ubuntu-latest
     steps:
       - uses: ika-rwth-aachen/docker-ros@main
         with:
+          platform: amd64,arm64
+          target: dev,run
           image-tag: ros
           base-image: rwthika/ros:noetic
           command: roslaunch etsi_its_conversion converter.ros1.launch
-          platform: amd64,arm64
-          target: dev,run
+          enable-industrial-ci: 'true'
           enable-recursive-vcs-import: 'false'
 
-  ros2:
+  ros2-humble:
     runs-on: ubuntu-latest
     steps:
       - uses: ika-rwth-aachen/docker-ros@main
         with:
+          platform: amd64,arm64
+          target: dev,run
           image-tag: ros2
           base-image: rwthika/ros2:humble
           command: ros2 launch etsi_its_conversion converter.launch.py
+          enable-industrial-ci: 'true'
+          enable-push-as-latest: 'true'
+          enable-recursive-vcs-import: 'false'
+  
+  ros2-iron:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ika-rwth-aachen/docker-ros@main
+        with:
           platform: amd64,arm64
           target: dev,run
+          image-tag: ros2-iron
+          base-image: rwthika/ros2:iron
+          command: ros2 launch etsi_its_conversion converter.launch.py
+          enable-industrial-ci: 'true'
           enable-push-as-latest: 'true'
           enable-recursive-vcs-import: 'false'
+
+  ros2-jazzy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ika-rwth-aachen/docker-ros@main
+        with:
+          platform: amd64,arm64
+          target: dev,run
+          image-tag: ros2-jazzy
+          base-image: ubuntu:24.04
```

Comment:
If the `base-image` doesn't contain the ROS installation, we should define the `ros-distro` arg. Alternatively, we could just use `ros:jazzy`, couldn't we?

Reply:
With existing jazzy images, I encountered this error:

```
root@828513742cf8:/# curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
root@828513742cf8:/# echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list
deb [arch=amd64 signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu noble main
root@828513742cf8:/# 
root@828513742cf8:/# apt update
E: Conflicting values set for option Signed-By regarding source http://packages.ros.org/ros2/ubuntu/ noble: /usr/share/keyrings/ros2-latest-archive-keyring.gpg != /usr/share/keyrings/ros-archive-keyring.gpg
E: The list of sources could not be read.
```

We might need to fix something in docker-ros. Will try to use ubuntu:24.04 for now.